composer/datasets/ade20k.py

```diff
@@ -314,7 +343,7 @@ def initialize_object(self, batch_size, dataloader_hparams) -> DataSpec:
                     RandomResizePair(min_scale=self.min_resize_scale,
                                      max_scale=self.max_resize_scale,
                                      base_size=(self.base_size, self.base_size)),
-                    RandomCropPair(crop_size=(self.final_size, self.final_size)),
+                    RandomCropPair(crop_size=(self.final_size, self.final_size), class_max_percent=0.75, n_retry=10),
```

Comment:
these should be mult-line. Pro tip: if you add a common after the last argument (e.g. `RandomCropPair(..., n_retry=10,)`, yapf will autoformat that for you with `make style`.

Also, we are moving to `num` instead of `n` prefixes, so can you make this `num_retry`?

Reply:
Done and Done!